docs/fundamentals/executor/executor-api.md

```diff
@@ -1,77 +1,14 @@
-(executor)=
-# Create Executor
+# Executor API
 
-{class}`~jina.Executor` process `DocumentArray` in-place via functions decorated with `@requests`. To create an Executor, you only need to follow three principles:
+{class}`~jina.Executor` process `DocumentArray` via functions decorated with `@requests`. To create an Executor, you only need to follow three principles:
 
 - An `Executor` should subclass directly from `jina.Executor` class.
-- An `Executor` class is a bag of functions with shared state (via `self`); it can contain an arbitrary number of
+- An `Executor` class is a bag of functions with shared state or configuration (via `self`); it can contain an arbitrary number of
   functions with arbitrary names.
 - Functions decorated by `@requests` will be invoked according to their `on=` endpoint. These functions can be coroutines (`async def`) or regular functions.
```

Comment:
```suggestion
2. An `Executor` class is a bag of functions with shared state or configuration (via `self`); it can contain an arbitrary number of
  functions with arbitrary names.
3. Functions decorated by `@requests` will be invoked according to their `on=` endpoint. These functions can be coroutines (`async def`) or regular functions.
```